File: POTD/7th_October_2024.java
class Solution {
    // function should insert the data to the front of the list
    

    // function to print the linked list
    static Node insert(Node head, int data) {
        // Code Here.
        
        Node new_head=new Node(data);
        new_head.npx = head;
        head=new_head;
        return head;
        
    }

    // function to print the linked list
    static ArrayList<Integer> getList(Node head) {
        // Code Here.
        ArrayList<Integer> arr = new ArrayList<>();
        while(head!=null){
            arr.add(head.data);
            head=head.npx;
        }
        return arr;
    }
}

File: POTD/7th_October_2024.cpp
struct Node *insert(struct Node *head, int data) {
    // Code here
    
    Node *new_head=new Node(data);
    new_head->npx=head;
    head=new_head;
    
}

vector<int> getList(struct Node *head) {
    // Code here
    
    vector<int>vec;
    
    while(head){
        vec.push_back(head->data);
        head=head->npx;
    }
    
    return vec;
    
}
